### PR TITLE
Changed formatting of README

### PR DESCRIPTION
**Pull request by ddavies90**
Check out the updated formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # odin-recipes
-Recipe website!
-First attempt at making a website from scratch
+
+## Recipe website!
```
Comment (KTraynor580) [inline README.md:3]: I prefer this formatting! 👍

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
+**First attempt at making a website from scratch**
```
Comment (KTraynor580) [inline README.md]: Not a big fan of the bold here

**Review by KTraynor580 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ddavies90) [reply README.md]: Bold removed

**Review by KTraynor580 — APPROVED** [review]
LGTM!

**Pull request merged**